app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,208 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this file even used? I didn't find usages anywhere. It also has the push notifications switches within it and those were moved to settings fragment.